src/components/SaveDelete/SaveDelete.js

```diff
@@ -1,42 +1,62 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import { confirmAlert } from 'react-confirm-alert';
-import 'react-confirm-alert/src/react-confirm-alert.css';
 import { connect } from 'react-redux';
+import 'react-confirm-alert/src/react-confirm-alert.css';
 import { deleteList, setUpdateStatus } from '../../redux/actions';
-
 import './SaveDelete.css';
 
 class SaveDelete extends PureComponent {
-
   handleUpdate = () => {
-    this.props.setUpdateStatus();
-    console.log('user in handleUpdate', this.props.user)
-    console.log('user in handleUpdate', this.props.user.username)
-    fetch(`/api/movies/save/${this.props.user.username}`, {
-      method: 'PUT',
-      headers: {
-        'Content-Type': 'application/json'
+    const {
+      setUpdateStatus,
+      user,
+      user: {
+        username,
       },
-      body: JSON.stringify(this.props.user)
-    })
-    .then(res => res.json())
-    .catch(err => console.log(err));
-  }
+    } = this.props;
+
+    setUpdateStatus();
+
+    console.log('user in handleUpdate', user);
+    console.log('user in handleUpdate', username);
+
+    fetch(`/api/movies/save/${username}`, {
```

Comment:
I wasn't sure why you were using both `axios` and `fetch`, so I left this here, however you'd want to also add this to `src/utils/http`